unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/UpdateEventHandler/UpdateDispatcher.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace DCL
+{
+    public class UpdateDispatcher : MonoBehaviour
```

Comment:
UpdateDispatcher sound as bitter misleading here. Since it will dispatch more calls than Update.  WDYT about  something like `MonoBevhaviourCallsDispatcher `